test/stresses.jl

```diff
@@ -0,0 +1,45 @@
+# Hellmann-Feynman stress
+# via ForwardDiff & custom FFTW overloads on ForwardDiff.Dual
+using Test
+using DFTK
+using ForwardDiff
+import FiniteDiff
+
+@testset "ForwardDiff stresses on silicon" begin
+    function make_basis(a)
+        lattice = a / 2 * [[0 1 1.];
+                        [1 0 1.];
+                        [1 1 0.]]
+        Si = ElementPsp(:Si, psp=load_psp(:Si, functional="lda"))
+        atoms = [Si => [ones(3)/8, -ones(3)/8]]
+        model = model_DFT(lattice, atoms, [:lda_x, :lda_c_vwn])
+        kgrid = [1, 1, 1] # k-point grid (Regular Monkhorst-Pack grid)
```

Comment:
Two spaces before `#` ... also these comments are sort of not needed for the tests. It's a pretty standard setup. Also we have data for standard test cases centralised in `testcases.jl` So you better use that. See the other tests for examples (search for `silicon.lattice`).

Reply:
I now added the re-use of `atnum`, `psp` and `positions` from silicon in testcases.jl